package.json

```diff
@@ -26,6 +26,7 @@
     "dev": "cross-env NODE_ENV=development node ./bin/build.js",
     "build": "cross-env NODE_ENV=production node ./bin/build.js",
     "lint": "eslint --ignore-path .gitignore ./src && prettier --check ./src",
+    "lint:fix": "eslint --ignore-path .gitignore ./src --fix",
```

Comment:
Looks good, what if we combine these two lines so that eslint automatically lint and auto-fixes and then executes prettier check?
```
    "lint": "eslint --ignore-path .gitignore ./src --fix && prettier --check ./src",
```

Reply:
We need a script for only checking because we don't want to fix stuff during CI, as the changes would not be saved in the repo.